### PR TITLE
Tag RigidBodySim.jl v0.1.0

### PR DESCRIPTION
**Pull request by attobot**
Repository: [tkoolen/RigidBodySim.jl](https://github.com/tkoolen/RigidBodySim.jl)
Release: [v0.1.0](https://github.com/tkoolen/RigidBodySim.jl/releases/tag/v0.1.0)
Travis: [Travis Build Status](https://travis-ci.org/tkoolen/RigidBodySim.jl/builds/347854805)
Diff: [vs v0.0.1](https://github.com/tkoolen/RigidBodySim.jl/compare/617b6be2fb30557de33a164520ec4511b300ad6d...e8f7124c73182ca045615168a6d3adfb70062e6e)
`requires` vs v0.0.1: 
```diff
--- 0.0.1/requires
+++ 0.1.0/requires
@@ -1,5 +1,5 @@
 julia 0.6
-RigidBodyDynamics 0.4.0
+RigidBodyDynamics 0.5.0
 OrdinaryDiffEq 3.0.1
 DiffEqCallbacks 1.0.0
 RigidBodyTreeInspector 0.3.2
```
cc: @tkoolen

Please make sure that:
- CI passes for supported Julia versions (if applicable).
- Version bounds reflect minimum requirements.

### CODE REVIEW

**Comment by tkoolen** [comment]
```
MethodError(Core.Inference.convert, (AssertionError, "invalid age range update"), 0x0000000000000ac6)
```
This is https://github.com/JuliaLang/julia/issues/21653. I'm getting this sporadically; I thought it was resolved, but apparently it isn't. It's very annoying.

**Comment by tkoolen** [comment]
I was finally able to work around the age range update failure. The remaining error occurs during `Pkg.build("DrakeVisualizer")`:
```
LoadError: failed process: Process(`sudo apt-get install libqt4-dev`, ProcessExited(100)) [100]
```
@rdeits, could you take a look?

**Comment by rdeits** [comment]
Right before that it says

```
E: Failed to fetch http://archive.ubuntu.com/ubuntu/pool/main/m/mesa/mesa-common-dev_17.2.4-0ubuntu1~16.04.4_amd64.deb  404  Not Found [IP: 91.189.88.161 80]
```

which sounds like it might just be an internet connection issue.

**Comment by rdeits** [comment]
Oh, no, that file really is missing. Ok, I'll investigate.

**Comment by rdeits** [comment]
I am able to build `DrakeVisualizer.jl` inside a clean Ubuntu 14.04 docker image with only `build-essential`, `pkg-config` and `unzip` installed. Interestingly, the *first* error in http://juliarun-ci.s3.amazonaws.com/d129116d7829bc80714148b00f397da6930e98cd/pull_request_of_RigidBodySim_on_julia_0_6.log shows up when RigidBodySim.jl tries to launch `drake-visualizer` from the command line *before* any `INFO: Building DrakeVisualizer` lines appear. So whatever is going wrong with the build is not even captured in the logfile. 

Basically, I can't tell what JuliaCIBot was doing, but I am pretty convinced that the DrakeVisualizer.jl build works. 

Independent of the build, though, DrakeVisualizer.jl's *tests* do require `xvfb` when running headless, so they're doomed to fail here unless there's some way for me to tell JuliaCIBot to use `xvfb`. 

In summary, we have this package working on Travis and locally, and I'd like to request that the JuliaCIBot issues be ignored for now.

**Comment by andreasnoack** [comment]
@rdeits Thanks for investigating. Quite a few packages now require `xvfb` so we might provide this feature eventually. For now, we'll just rely on passing Travis tests and ignore CIBot here.
